qiita_db/processing_job.py

```diff
@@ -741,7 +741,7 @@ def _generate_notification_message(self, value, error_msg):
                        f'{self.command.parameters}')
         else:
             for artifact in input_artifacts:
-                if artifact.prep_templates is not None:
+                if artifact.prep_templates:
```

Comment:
yup, that was the main error being solved: artifact.prep_templates is a list and if nothing is found is an empty list AKA always not None. This wasn't originally caught because there was no test for the else.